es-core/src/components/ComponentList.cpp

```diff
@@ -319,7 +319,7 @@ std::vector<HelpPrompt> ComponentList::getHelpPrompts()
 		bool addMovePrompt = true;
 		for(auto it = prompts.begin(); it != prompts.end(); it++)
 		{
-			if(it->first == "up/down" || it->first == "up/down/left/right")
+			if(strcmp(it->first, "up/down") ==0 || strcmp(it->first, "up/down/left/right") == 0)
```

Comment:
Apologies for the nitpick: missing space between == and 0, if you have the chance. :/